packages/framework/src/Model/Administrator/Security/AdministratorFrontSecurityFacade.php

```diff
@@ -40,22 +42,56 @@ class AdministratorFrontSecurityFacade
      */
     protected $authorizationChecker;
 
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain|null
+     */
+    protected ?Domain $domain;
```

Comment:
`protected ?Domain $domain = null`? To avoid initialized property in extended class? :thinking: